mne/viz/misc.py

```diff
@@ -346,6 +365,10 @@ def plot_bem(subject=None, subjects_dir=None, orientation='coronal',
         'coronal' or 'axial' or 'sagittal'.
     slices : list of int
         Slice indices.
+    brain_surfaces : None | str | list of str
+        One or more brain surface to plot (optional). Entries should correspond
+        to files in the subject's ``surf`` directory, e.g., ``"white"`` will
+        plot ``<subject>/surf/lh.white`` and ``<subject>/surf/rh.white``.
```

Comment:
would be useful especially for deep structures. Can you check that it works too for a volume or discrete source space?